### ISSUE TITLE
508: Failed login message box

### ISSUE THREAD

**Issue by annalbrecht** [US-EPA-CAMD/PI-1#6832]
The validation error message box for a failed login includes an X button to dismiss the message. This button is not keyboard accessible (set to aria-hidden: true). Maybe remove the tabindex of -1.

Only the X button should be actionable, not the entire message.

<img width="975" height="173" alt="Image" src="https://github.com/user-attachments/assets/8c813a39-91ca-4813-9902-f74beb14da4f" />

Developer Notes: 
Update Message Tab. Only the X button should be clickable. 
aria-live = "assertive" + aria-atomic = "true" ensures screen readers read the message right away and makes screen reader read the whole message. 
Add fucus() to message container to allow screen reader focus. 

Acceptance Criteria: 
Text is accessible to screen readers
Error message should be actionable only when X button is clickable. 
Keyboard accessible will be fix with ticket https://github.com/US-EPA-CAMD/easey-ui/issues/6834

**Comment by maxdiebold-erg** [comment]
Tested in **DEV**.

It is now possible to Tab to the banner "X" and close it with the Spacebar.

<img width="1270" height="102" alt="Image" src="https://github.com/user-attachments/assets/d561120a-3dbe-4afa-a650-90dccec342da" />

Also confirmed that no errors are reported by the WAVE tool. The ANDI tool does report a couple warnings in the banner with the following message:

> Element is focusable but has or is contained by [aria-hidden=true].

This is referencing elements with `tabindex="-1"` and `aria-hidden=true`, which are programmatically focusable but not tab-able, so I think this can be disregarded.

**These changes need to be propagated to the warning banner as well:**

<img width="1559" height="148" alt="Image" src="https://github.com/user-attachments/assets/acd0e900-b644-4e0a-8117-9ae5eb5561b9" />

**Comment by maxdiebold-erg** [comment]
I'm now able to Tab to the close button of the warning banner, too.

<img width="1576" height="154" alt="Image" src="https://github.com/user-attachments/assets/32088ebf-ed68-4ad1-83a1-49b1c5a576f4" />

**Comment by maheese** [comment]
Confirmed in perf.
